cibuildwheel/util.py

```diff
@@ -184,6 +188,17 @@ def auto_archs(platform: PlatformName) -> 'Set[Architecture]':
             result.add(Architecture.x86)
         return result
 
+    @staticmethod
+    def all_archs(platform: PlatformName) -> 'Set[Architecture]':
```

Comment:
Does this overlap with `ALLOWED_ARCHITECTURES`? In values, definitely. But conceptually as well?

Reply:
Yes, they should be the same. Good catch!